src/pages/Resume/EmploySearchDetailPage.test.tsx

```
@@ -0,0 +1,246 @@
+      mockUseGetApplicantResume.mockReturnValue({
+        data: {
+          is_scraped: true,
+          data: {
+            personal_information: {
+              phone_number: '010-1234-5678',
+            },
+          },
+        },
+        isLoading: false,
+        error: null,
+      } as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)